src/cli/global/remove.rs

```diff
@@ -34,13 +34,16 @@ impl HasSpecs for Args {
 }
 
 pub async fn execute(args: Args) -> miette::Result<()> {
+    let Some(env_name) = &args.environment else {
+        miette::bail!("`--environment` is required. Try `pixi global uninstall` if you want to delete the whole environment");
```

Comment:
Could even include the `packages` names to make the command directly runnable.

Reply:
Great idea, done!